examples/feature_graph_backed_assets/feature_graph_backed_assets/definitions.py

```diff
@@ -0,0 +1,24 @@
+from dagster import (
+    AssetsDefinition,
+    AssetSelection,
+    Definitions,
+    define_asset_job,
+    load_assets_from_package_module,
+)
+
+from . import assets
+from .graphs_and_ops import layover_breakdown_2022, us_assets
+
+airline_job = define_asset_job(
+    "airline_job", AssetSelection.assets("passenger_flights").downstream()
+)
+
+
+defs = Definitions(
+    assets=[
+        *load_assets_from_package_module(assets),
+        AssetsDefinition.from_graph(us_assets),
+        AssetsDefinition.from_graph(layover_breakdown_2022),
+    ],
+    jobs=[define_asset_job("airline_job", AssetSelection.assets("passenger_flights").downstream())],
+)
```

Comment:
@maximearmstrong pre-existing problem but `airline_job` variable appears unused and pointless?

Reply:
Good catch - I missed that when moving things around. I will remove it in my second PR.